chain.schema.json

```diff
@@ -169,6 +169,11 @@
         "recommended_version": {
           "type": "string"
         },
+        "go_version": {
+          "type": "string",
+          "pattern": "^[0-9]+\\.[0-9]+\\.[0-9]+$",
```

Comment:
change to `^[0-9]+\\.[0-9]+\\.[0-9]?$` so as to match both `1.19` and `1.19.1` as an example.

Reply:
Oh dang, you're right. I thought new versions ended with `.0`. Thanks!

<img width="388" alt="Screenshot 2023-12-07 at 6 13 39 AM" src="https://github.com/cosmos/chain-registry/assets/9121234/60fd2ab4-43cb-4395-9efd-f45b0639052e">

Reply:
Revised, this is the new regex:

```
^[0-9]+\\.[0-9]+(\\.[0-9]+)?$
```